news/2020-03-13-featured-artist-beatmap-updates-from-the-mappers-guild.md

```diff
@@ -0,0 +1,439 @@
+- [KNOWER - BUTTS TITS MONEY](https://osu.ppy.sh/beatmapsets/1076064#osu/2251648) hosted by [Imakuri](https://osu.ppy.sh/users/6100837) (osu!)
+
+![Kola Kid header](https://assets.ppy.sh/artists/67/header.jpg)
+
+For the **Kola Kid short song mini-pack (osu!mania)** quest, the mappers had to create and rank one mapset for each Kola Kid songs with a length below 1:00, each hosted by a different user.
+
+This quest was completed by **[Unpredictable](https://osu.ppy.sh/users/7560872)**, **[Hydria](https://osu.ppy.sh/users/808176)** and **[Feerum](https://osu.ppy.sh/users/4815717)**
+
+- [Kola Kid - sea shells](https://osu.ppy.sh/beatmapsets/1114940#mania/2329040) hosted by [Feerum](https://osu.ppy.sh/users/4815717)
+- [Kola Kid - press start](https://osu.ppy.sh/beatmapsets/949646#mania/1983010) by [Hydria](https://osu.ppy.sh/users/808176)
+- [Kola Kid - broken legs](https://osu.ppy.sh/beatmapsets/1116952#mania/2333287) hosted by [Unpredictable](https://osu.ppy.sh/users/7560872)
+
+![Loki header](https://assets.ppy.sh/artists/7/header.jpg)
+
+For the **Loki's "Tales of the Mountains" (osu!)** quest, the mappers had to create and rank a mapset for each song in Loki's concept album "Tales of the Mountains", each hosted by a different user.
+
+This quest was completed by **[Cris-](https://osu.ppy.sh/users/6175280)**, **[walaowey](https://osu.ppy.sh/users/1475828)**, **[Minorsonek](https://osu.ppy.sh/users/5377178)**, **[Cosmolade](https://osu.ppy.sh/users/2172778)**, **[Gero](https://osu.ppy.sh/users/1467715)**, **[Tatan](https://osu.ppy.sh/users/5646529)**, **[Crissa](https://osu.ppy.sh/users/5405836)**, **[pkhg](https://osu.ppy.sh/users/343624)**, **[Seolv](https://osu.ppy.sh/users/8067876)** and **[TheShadowOfDark](https://osu.ppy.sh/users/5795337)**
+
+- [Loki - Quiet Village](https://osu.ppy.sh/beatmapsets/1050693#osu/2195976) hosted by [Gero](https://osu.ppy.sh/users/1467715)
+- [Loki - Wizard's Tower](https://osu.ppy.sh/beatmapsets/1064248#osu/2228263) hosted by [Minorsonek](https://osu.ppy.sh/users/5377178)
+- [Loki - Troll of the Forsaken Cave](https://osu.ppy.sh/beatmapsets/1052028#osu/2198578) hosted by [Cris-](https://osu.ppy.sh/users/6175280)
+- [Loki - With Fire and Sword (2013)](https://osu.ppy.sh/beatmapsets/1088528#osu/2276018) hosted by [TheShadowOfDark](https://osu.ppy.sh/users/5795337)
+- [Loki - The Victory is Ours](https://osu.ppy.sh/beatmapsets/1066365#osu/2232865) hosted by [Crissa](https://osu.ppy.sh/users/5405836)
+- [Loki - Interlude](https://osu.ppy.sh/beatmapsets/1091243#osu/2281346) hosted by [Tatan](https://osu.ppy.sh/users/5646529)
+
+![Mystery header](/wiki/shared/news/banners/mappersguild-mystery.jpg)
+
+For the **Mystery Electronic mini-pack (osu!taiko)** quest, the mappers had to create and rank at least 3 mapsets of unpublished electronic songs by one of our unreleased featured artists.
```

Comment:
insert paragraph about how these maps were paired with the fa announcement for MYLK https://osu.ppy.sh/home/news/2019-12-31-new-featured-artist-mylk